### PR TITLE
fix: reset wallet

### PR DESCRIPTION
**Pull request by r4mmer**
### Acceptance Criteria
- Allow the wallet to be reset successfully

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

**Review by tuliomir — APPROVED** [review]
I feel the `this.clearItems(true)` on line 174 should already do what this new lines do.

But since this is an internal method within the `store`, it's ok to have bits of redundance. Or would it be a better idea to use the `clearItems` method to also clean these `walletKeys`?

**Comment by r4mmer** [comment]
>I feel the this.clearItems(true) on line 174 should already do what this new lines do.
>
>But since this is an internal method within the store, it's ok to have bits of redundance. Or would it be a better idea to use the clearItems method to also clean these walletKeys?

The `resetWallet` method objective is to clear the wallet keys, so we can create a new wallet without losing other configuration (e.g. server, network, etc.), while `clearItems` is meant to erase all keys. The true argument is so we clear keys only from the legacy storage, this way we can reset a wallet from the locked screen even if the user did not unlock after the upgrade.

File: src/store.js [hunk 1]
```diff
@@ -166,6 +166,10 @@ class AsyncStorageStore {
   async resetWallet() {
     // This should delete the access data and registered tokens, the only persisted data
     await AsyncStorage.multiRemove(walletKeys);
+    // Delete the same keys on the memory cache
```
Comment (pedroferreira1) [inline src/store.js:169]: I feel the `await this.clearItems(true);` line below is already deleting those keys, isn't it?

Comment (pedroferreira1) [reply src/store.js:169]: Actually it's not but it's confusing. `onlyWalletKeys` and `walletKeys` don't mean the same thing

Comment (r4mmer) [reply src/store.js:169]: Yes, onlyWalletKeys mean the legacy wallet keys which are prefixed with wallet:

**Review by pedroferreira1 — APPROVED** [review]

**Pull request merged**